.github/actions/javascript/reviewerChecklist/index.js

```diff
@@ -62,6 +71,7 @@ function checkIssueForCompletedChecklist(numberOfChecklistItems) {
 
                 // Found the reviewer checklist, so count how many completed checklist items there are
                 if (comment.startsWith(reviewerChecklistStartsWith)) {
+                    console.log('Found the reviewer checklist!');
                     foundReviewerChecklist = true;
                     numberOfFinishedChecklistItems = (comment.match(/- \[x\]/gi) || []).length;
                     numberOfUnfinishedChecklistItems = (comment.match(/- \[ \]/g) || []).length;
```

Comment:
Let's add a new early return right after looping through the comments which is like:

```
if (!foundReviewerChecklist) {
    core.setFailed('No PR Reviewer Checklist was found');
    return;

}
```